tests/unit_test/log_test/log_processor_test.py

```
@@ -90,6 +90,263 @@ class DummyRequest:
+        page_size=10,
+        **(kwargs or {})
+    )
+
+def test_default_from_to_dates_for_model_testing_logs():
+    handler = mock_model_testing_handler()
+    handler.doc_type.objects = MagicMock()
+
+    try:
+        handler.get_logs_and_count()
+    except Exception:
+        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)